kong/router/atc.lua

```diff
@@ -67,7 +67,10 @@ do
                   "http.queries.*",
                  },
 
-    ["Int"]    = {"net.port",
+    ["Int"]    = {"net.src.port", "net.dst.port",
+                 },
+
+    ["IpAddr"] = {"net.src.ip", "net.dst.ip",
```

Comment:
What kind of impact will this have on version compatibility in hybrid mode for older data planes?

Reply:
This PR did not change the expression stored in db, only convert it adhoc, so if dp does not upgrade to the latest version, it will still work well.

Reply:
@mikefero Will send you a PM